src/sentry/search/utils.py

```diff
@@ -472,7 +472,7 @@ def get_terminal_release_for_group(
     group: Group,
     query_type: LatestReleaseOrders,
     latest: bool,
-) -> Sequence[str]:
+) -> Release:
```

Comment:
FYI This file is excluded from coverage
https://github.com/getsentry/sentry/blob/b1220adca50d9cc9ae9ff275098425c1d3e29148/pyproject.toml#L537